### DISCUSSION TITLE
read HTTP REST

### DISCUSSION THREAD

**Discussion by ErfanDL** [grafana/grafana-json-datasource#49] [Q&A]
Hi can I read value from HTTP REST API ? 
http://10.5.51.5:8080/1ba32e9c834340ed83a795c083c03c4a/get/v27
how can I read this URL ?

thanks.

**Comment by marcusolsson** [comment]
You should be able to enter the URL in the data source config, as is. If you're having trouble connecting, could you share some more details on the error your getting?
